### ISSUE TITLE
Multiple Project Management

### ISSUE THREAD

**Issue by MRC38** [bryntum/support#4013]
We use Bryntum Gantt to manage single independent projects, but also we manage multiple projects under a 'programme'.  So in effect a large project consisting of many smaller projects,  see image below.

Do you support this kind of arrangement, or have plans to?

Currently this gives us many issues.  If we have a 'programme' start date and links between projects, when we look at the projects individually we often get different dates to when we look at the same project within the 'programme'.  Also when switching between the project view and the programme view we get continuous Gantt updates.

![image](https://user-images.githubusercontent.com/34025208/149302994-fc243aaa-9929-47bf-9bce-7dcfc8c1ba66.png)

**Comment by MRC38** [comment]
We have implemented a programme approach [cluster of projects], but there is a hurdle to overcome between Project Start date and Constraint dates when viewing a project separately from its programme.  

You have labelled this a feature request, have you assessed this and are you including in your roadmap?

**Comment by matsbryntse** [comment]
We have not yet discussed internally, but we will update this issue once we have - please stay tuned!

**Comment by vloax** [comment]
any update on this?

**Comment by matsbryntse** [comment]
Not yet. @MRC38 can you please share a small test case where we can see the issues you mentioned?

**Comment by vloax** [comment]
the current solution to support multiple projects is to use the projects like a task? 

what do i do with the global gantt/scheduler configuration, like calendar, resources and etc.. that changes based on the project

**Comment by matsbryntse** [comment]
> the current solution to support multiple projects is to use the projects like a task?
Yes that's currently the only way to address this. We aim to revisit this topic as part of our future roadmap.
